client-ts/FunctionalTests/FunctionalTests.csproj

```diff
@@ -25,6 +37,18 @@
     <Folder Include="wwwroot\js\" />
   </ItemGroup>
 
+  <ItemGroup>
+    <TypeScriptCompile Include="selenium\run-ci-tests.ts" />
+    <TypeScriptCompile Include="selenium\run-tests.ts" />
+    <TypeScriptCompile Include="ts\Common.ts" />
+    <TypeScriptCompile Include="ts\ConnectionTests.ts" />
+    <TypeScriptCompile Include="ts\HubConnectionTests.ts" />
+    <TypeScriptCompile Include="ts\index.ts" />
+    <TypeScriptCompile Include="ts\Utils.ts" />
+    <TypeScriptCompile Include="ts\WebDriverReporter.ts" />
+    <TypeScriptCompile Include="ts\WebSocketTests.ts" />
+  </ItemGroup>
```

Comment:
Interesting, @mikeharder saw this happen too... I haven't been able to figure out what causes it yet :(

Reply:
Newer VS?

Reply:
Can you revert before merging? I'd like to avoid having it in our project.